packages/components/src/components/tag/_tag.scss

```diff
@@ -195,7 +200,7 @@
   .#{$prefix}--tag__close-icon:focus {
     border-radius: 50%;
     outline: none;
-    box-shadow: inset 0 0 0 2px $focus;
+    box-shadow: inset 0 0 0 1px $focus;
```

Comment:
Do we also need to change this on L214?

Reply:
I kept it since it was high contrast specific but I'll reduce it for consistency

Reply:
Yeah just wasn't sure if it was intentional or not, just wanted to bring it up 👍